src/engine/fileOpsIns.cpp

```diff
@@ -444,13 +448,17 @@ void DivEngine::loadS3I(SafeReader& reader, std::vector<DivInstrument*>& ret, St
       // Skip more stuff we don't need
       reader.seek(21, SEEK_CUR);
     } else {
-      logE("S3I PCM samples currently not supported.");
+      lastError = "S3I PCM samples currently not supported.";
+      logE("S3I PCM samples currently not supported.\n");
```

Comment:
`\n` is no longer needed after the log system rewrite.

Reply:
Done. Also opted to normalise duplicate strings.